### DISCUSSION TITLE
手机访问博客就没有顶部的模糊效果，这个是在哪里配置的吗

### DISCUSSION THREAD

**Discussion by chendada00** [anzhiyu-c/hexo-theme-anzhiyu#216] [General]
![image](https://github.com/anzhiyu-c/hexo-theme-anzhiyu/assets/53457984/0abfc5ce-4115-4237-bdd1-978311be7222)

**Comment by wwzhishi** [comment]
同一问题，你解决了吗
![png](https://github.com/user-attachments/assets/b76c912b-42f2-4f00-8595-572f072c3d32)

**Comment by wwzhishi** [comment]
解决了，打开获取文章主色调就可以了，[主题文档](https://docs.anheyu.com/advanced/#%E4%B8%BB%E9%A2%98%E5%A6%82%E4%BD%95%E8%8E%B7%E5%8F%96%E6%96%87%E7%AB%A0%E4%B8%BB%E8%89%B2%E8%B0%83)
如图
![png](https://github.com/user-attachments/assets/366fd02e-c64c-4bf4-8bd2-243eb385c836)
